### ISSUE TITLE
When scrolling with keys and then with mouse, qutebrowser jumps up

### ISSUE THREAD

**Issue by magnetophon** [qutebrowser/qutebrowser#3748]
If you first scroll down with this key binding:
``config.bind('j', 'scroll down')``
and then scroll with the mousewheel, qutebrowser jums to the top of the page and scrolls from there.

qutebrowser v1.2.0
Git commit: 
Backend: QtWebEngine (Chromium 61.0.3163.140)

CPython: 3.6.4
Qt: 5.10.1
PyQt: 5.10

sip: 4.19.6
colorama: no
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: 1.0.1 $Id$
attr: 17.4.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: no
sqlite: 3.21.0
QtNetwork SSL: OpenSSL 1.0.2n  7 Dec 2017

Style: QFusionStyle
Platform: Linux-4.15.7-x86_64-with-glibc2.3.4, 64bit
Linux distribution: NixOS 18.03.git.a619890 (Impala) (unknown)
Frozen: False
Imported from /nix/store/h8j3b3la9pd0j2a4z5cx1w8k24371jkh-qutebrowser-1.2.0/lib/python3.6/site-packages/qutebrowser
Qt library executable path: /nix/store/cv836804w0lcbmpan1aljzlzq6v3f66z-qtbase-5.10.1/libexec, data path: /nix/store/cv836804w0lcbmpan1aljzlzq6v3f66z-qtbase-5.10.1
OS Version: 

--- /etc/os-release ---
NAME=NixOS
ID=nixos
VERSION="18.03.git.a619890 (Impala)"
VERSION_CODENAME=impala
VERSION_ID="18.03.git.a619890"
PRETTY_NAME="NixOS 18.03.git.a619890 (Impala)"

Paths:
cache: /home/bart/.cache/qutebrowser
config: /home/bart/.config/qutebrowser
data: /home/bart/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser

**Comment by The-Compiler** [comment]
Does this always happen or only after opening a new tab (#395)? Do you have a hidden statusbar (#2236)?

**Comment by magnetophon** [comment]
I am not able to reproduce it now, sorry.
When trying to reproduce it, I did notice that instead of scrolling with ``j``, I am probably using ``SPC``.

I do not have a hidden statusbar.

**Comment by The-Compiler** [comment]
I'm guessing this is somehow related to opening tabs then, i.e. the same as #395. If you find out more, let me know, though!

**Comment by The-Compiler** [comment]
Duplicate of #395
